src/main/java/com/stripe/param/AccountCreateParams.java

```diff
@@ -0,0 +1,2587 @@
+// Generated by com.stripe.generator.entity.SdkBuilder
+
+package com.stripe.param;
+
+import com.google.gson.annotations.SerializedName;
+import com.stripe.net.ApiRequestParams;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import lombok.Getter;
+
+@Getter
+public class AccountCreateParams extends ApiRequestParams {
+  /**
+   * An [account token](https://stripe.com/docs/api#create_account_token), used to securely provide
+   * details to the account.
+   */
+  @SerializedName("account_token")
+  String accountToken;
+
+  /** Non-essential business information about the account. */
+  @SerializedName("business_profile")
+  BusinessProfile businessProfile;
+
+  /** The business type. Can be `individual` or `company`. */
+  @SerializedName("business_type")
+  String businessType;
+
+  /**
+   * Information about the company or business. This field is null unless `business_type` is set to
+   * `company`.
+   */
+  @SerializedName("company")
+  Company company;
+
+  /**
+   * The country in which the account holder resides, or in which the business is legally
+   * established. This should be an ISO 3166-1 alpha-2 country code. For example, if you are in the
+   * United States and the business for which you're creating an account is legally represented in
+   * Canada, you would use `CA` as the country for the account being created.
+   */
+  @SerializedName("country")
+  String country;
+
+  /**
+   * Three-letter ISO currency code representing the default currency for the account. This must be
+   * a currency that [Stripe supports in the account's country](https://stripe.com/docs/payouts).
+   */
+  @SerializedName("default_currency")
+  String defaultCurrency;
+
+  /**
+   * The email address of the account holder. For Custom accounts, this is only to make the account
+   * easier to identify to you: Stripe will never directly email your users.
+   */
+  @SerializedName("email")
+  String email;
+
+  /** Specifies which fields in the response should be expanded. */
+  @SerializedName("expand")
+  List<String> expand;
+
+  /**
+   * A card or bank account to attach to the account. You can provide either a token, like the ones
+   * returned by [Stripe.js](https://stripe.com/docs/stripe.js), or a dictionary, as documented in
+   * the `external_account` parameter for [bank
+   * account](https://stripe.com/docs/api#account_create_bank_account) creation. <br>
+   * <br>
+   * By default, providing an external account sets it as the new default external account for its
+   * currency, and deletes the old default if one exists. To add additional external accounts
+   * without replacing the existing default for the currency, use the bank account or card creation
+   * API.
+   */
+  @SerializedName("external_account")
+  String externalAccount;
+
+  @SerializedName("from_recipient")
+  String fromRecipient;
```

Comment:
This has been undocumented for a while, why is it appearing here when others undocumented things don't appear?

Reply:
addressed in https://github.com/stripe/stripe-java/pull/711/commits/760b5b5713915de21ce7368dace7e88e9c5dc85f#diff-a9f0113cf10a4a96bc61507995cc5a57R62